code/game/objects/items/weapons/tools/saws.dm

```diff
@@ -53,7 +54,7 @@
 	icon_state = "chainsaw"
 	hitsound = WORKSOUND_CHAINSAW
 	worksound = WORKSOUND_CHAINSAW
-	force = WEAPON_FORCE_ROBUST
+	force = 33 // Rip and tear!
```

Comment:
Use WEAPON_FORCE_ROBUST * 1.2 or some new defined value.

Reply:
New defines created for the chainsaw and energy axe. Use them as a basis for creating similar weapons in the future (vibroswords? chain axes? _chainswords_?)